File: backend/src/main/java/com/tjh/riskfactor/entity/SaveGuard.java
package com.tjh.riskfactor.entity;

import lombok.Data;
import lombok.experimental.Accessors;

import javax.persistence.*;

@Data @Entity
@Table(name = "save_guard")
@Accessors(chain = true)
public class SaveGuard {

    @Id
    private Integer id;

}

File: backend/src/main/kotlin/com/tjh/riskfactor/entity/SaveGuard.kt
package com.tjh.riskfactor.entity

import javax.persistence.*

/**
 * 复杂动作的“已执行”flag，仅在[com.tjh.riskfactor.component.InitialDataLoader]中使用
 */
@Entity @Table(name = "save_guard")
class SaveGuard(@Id val id: Int = 0)
